jest.config.js

```diff
@@ -11,6 +11,16 @@ const suiteParams = {
         testMatch: ['**/*/*.screen.ts'],
         testRunner: 'jest-jasmine2',
     },
+    unit: {
+        globalSetup: './test/global-setup.js',
+        globalTeardown: './test/global-teardown.js',
+        maxWorkers: 5,
+        maxConcurrency: 3,
+        testTimeout: 15000,
+        testEnvironment: 'node',
+        testMatch: ['**/units/*.ts'],
```

Comment:
```suggestion
        testMatch: ['src/test/units/**/*.ts'],
```
Раз уж решил все юниты в одну папку вытащить